plugins/orchestrator/src/components/WorkflowDefinitionsListComponent/WorkflowDefinitionsListComponent.tsx

```diff
@@ -90,10 +145,14 @@ export const WorkflowsTable = ({ items }: WorkflowsTableProps) => {
 
   return (
     <Table
-      title=""
-      options={{ search: true, paging: false, actionsColumnIndex: 1 }}
+      options={{ search: true, paging: false, actionsColumnIndex: 5 }}
       columns={columns}
       data={data}
+      filters={[{ column: 'Type', type: 'select' }]}
+      initialState={{
+        filtersOpen: true,
+        filters: { Type: WorkflowType.ASSESSMENT },
```

Comment:
If I don't have workflows with the type ASSESSMENT, I see an empty table when I go to /orchestrator.
I think the filter type can be a state. We can set ASSESSMENT as the filter type only if there is at least one Assessment. Otherwise, maybe use "All results" or "Infrastructure.

<img width="1122" alt="image" src="https://github.com/caponetto/backstage-plugins/assets/638737/28ff5950-6a54-4979-97d5-be618c665ac7">

Reply:
ok we can display all workflows if assessment is empty